src/api/java/com/minecolonies/api/compatibility/CompatibilityManager.java

```diff
@@ -262,7 +262,8 @@ public boolean isCompost(final ItemStack itemStack)
     @Override
     public boolean isPlantable(final ItemStack itemStack)
     {
-        if (itemStack.isEmpty() || itemStack.getItem() == Items.WITHER_ROSE)
```

Comment:
Lol, I just checked into the config file and actually the reason it's here in the first place is because it's in the list of plantables in the default config. So just removing it there should solve this for the future already.